charts/stfc-cloud-openstack-cluster/README.md

```diff
@@ -5,28 +5,111 @@ This chart deploys a Cluster-API (CAPI) K8s Cluster intended to be run on the ST
+
+2. Run `cloud-helm-charts/charts/stfc-cloud-openstack-cluster/scripts/set-project-id.sh </path/to/clouds.yaml>`
+- This modifies your clouds.yaml to add the corresponding `project_id` to the appplication credential
+
+> [!NOTE] 
+> Step 2 is mandatory, but only needs to be run once per new clouds.yaml
+
+
+2. Run `cloud-helm-charts/charts/stfc-cloud-openstack-cluster/scripts/bootstrap.sh`
+
+> [!NOTE]
+> This script will setup a microk8s cluster which will be used to spin up your cluster 
+
+> [!NOTE]
+> This step is OPTIONAL if you 
+>   - have a bootstrap cluster already setup
+>   - you're spinning up a child cluster
 
-4. Install the chart
+3. Install the chart, `openstack-cluster.apiServer.floatingIP` needs to be set to the floating IP you setup to access your cluster - see Requirements
 
 ```bash
 export CLUSTER_NAME="demo-cluster"  # or your cluster name
-helm upgrade $CLUSTER_NAME cloud-charts/stfc-cloud-openstack-cluster --install -f values.yaml -f user-values.yaml -f flavors.yaml -f /tmp/capi/secret-values -n clusters 
+helm upgrade $CLUSTER_NAME cloud-charts/stfc-cloud-openstack-cluster --install -f values.yaml -f addons.yaml -f nodes.yaml -f /path/to/clouds.yaml --set openstack-cluster.apiServer.floatingIP=130.246.xxx.xxx
 ```
-5. Perform move to self-managed cluster 
 
-see https://stfc.atlassian.net/wiki/spaces/CLOUDKB/pages/211878034/Cluster+API+Setup#Deploying-Cluster
\ No newline at end of file
+4. Check the cluster status
+
+When the deployment is complete clusterctl will report the cluster as Ready: True
+
+```bash
+clusterctl describe cluster $CLUSTER_NAME -n clusters
+```
+
+Progress can be monitored with the following command in a separate terminal:
+
+```bash
+kubectl logs deploy/capo-controller-manager -n capo-system -f
+```
+
+Once this is deployed you can get the kubeconfig like so:
+
+```bash
+clusterctl get kubeconfig $CLUSTER_NAME -n clusters > $CLUSTER_NAME.kubeconfig
+```
+
+you should be able to run commands on your cluster like this:
+```bash
+KUBECONFIG=$CLUSTER_NAME.kubeconfig kubectl get nodes
+```
+
+5. (Optional) Perform move to self-managed cluster
+
+> [!NOTE]
+> Ignore this if you are deploying a child cluster
+
+Install clusterctl into the new cluster and move the control plane
+```bash
+clusterctl init --infrastructure=openstack:v0.10.5 --kubeconfig=$CLUSTER_NAME.kubeconfig
+clusterctl move --to-kubeconfig $CLUSTER_NAME.kubeconfig -n clusters
+```
+ 
+Ensure the control plane is now running on the new cluster:
+
+```bash
+kubectl get kubeadmcontrolplane --kubeconfig=$CLUSTER_NAME.kubeconfig -n clusters
+```
+
+Using the new control plane by default, 
+you can optionally replace the existing kubeconfig with the new cluster’s kubeconfig to make things easier
+
+```bash
+cp -v $CLUSTER_NAME.kubeconfig ~/.kube/config
+# Ensure kubectl now uses the new kubeconfig displayed the correct nodes:
+```
```

Comment:
I think most of this file duplicates our user facing docs: https://stfc.atlassian.net/wiki/spaces/CLOUDKB/pages/211878034/Kubernetes

Can we link them onto that from here instead that way we have a single "source-of-truth" and any improvements / changes don't need to be duplicated

I'm happy to move these docs (i.e. with the new `user-values.yaml` to the docs) after this PR since we can change the URLs at the same time

(On that note we need docs on getting users to migrate existing values files into this new format too in the Upgrades page when we sunset the other repo)

Reply:
I am keen to do that - but there's a few changes - mainly that `helm upgrade` step at the end which is different.  
I don't want to confuse people by pointing to out-of-date docs just yet. 

Duplicating it seems to be the easiest way to avoid issues

Reply:
Also I'd want to test this internally for now - have a few grads/apprentices try building clusters using this method before we migrate docs to public facing ones